Microsoft.Toolkit.Uwp.UI.Animations/ConnectedAnimations/Connected.cs

```diff
@@ -0,0 +1,426 @@
+        /// </summary>
+        /// <param name="obj">The <see cref="UIElement"/></param>
+        /// <returns>The anchor <see cref="UIElement"/></returns>
+        public static UIElement GetAnchorElement(DependencyObject obj)
+        {
+            return (UIElement)obj.GetValue(AnchorElementProperty);
+        }
+
+        /// <summary>
+        /// Sets the <see cref="UIElement"/> that is the anchor for the coordinated connected animation
+        /// </summary>
+        /// <param name="obj">The <see cref="UIElement"/> that will follow the anchor element</param>
+        /// <param name="value">The <see cref="UIElement"/> that should be followed</param>
+        public static void SetAnchorElement(DependencyObject obj, UIElement value)
+        {
+            obj.SetValue(AnchorElementProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the connected animation key associated with the ListViewBase item being animated
+        /// </summary>
+        /// <param name="obj">The <see cref="ListViewBase"/></param>
+        /// <returns>The connected animation key</returns>
+        public static string GetListItemKey(DependencyObject obj)
+        {
+            return (string)obj.GetValue(ListItemKeyProperty);
+        }
+
+        /// <summary>
+        /// Sets the connected animation key for the <see cref="ListViewBase"/> item being animated
+        /// </summary>
+        /// <param name="obj">The <see cref="ListViewBase"/></param>
+        /// <param name="value">The connected animation key</param>
+        public static void SetListItemKey(DependencyObject obj, string value)
+        {
+            obj.SetValue(ListItemKeyProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the name of the element in the <see cref="DataTemplate"/> that is animated
+        /// </summary>
+        /// <param name="obj">The <see cref="ListViewBase"/></param>
+        /// <returns>The name of the element being animated</returns>
+        public static string GetListItemElementName(DependencyObject obj)
+        {
+            return (string)obj.GetValue(ListItemElementNameProperty);
+        }
+
+        /// <summary>
+        /// Sets the name of the element in the <see cref="DataTemplate"/> that is animated
+        /// </summary>
+        /// <param name="obj">The <see cref="ListViewBase"/></param>
+        /// <param name="value">The name of the element to animate</param>
+        public static void SetListItemElementName(DependencyObject obj, string value)
+        {
+            obj.SetValue(ListItemElementNameProperty, value);
+        }
+
+        /// <summary>
+        /// Identifies the Connected.Key XAML attached property
+        /// </summary>
+        public static readonly DependencyProperty KeyProperty =
+            DependencyProperty.RegisterAttached("Key", typeof(string), typeof(Connected), new PropertyMetadata(null, OnKeyChanged));
```

Comment:
`Key` should use `nameof(Key)` (same for DPs below)

Reply:
How does this work for Attached Properties?

Reply:
D'oh! Good point.